process_model/process_model.py

```diff
@@ -1,72 +1,154 @@
-import click
-import re
+"""process_model outputs a Fortran module to interface to the given TF models
```

Comment:
Uber pedantic: one-line docstrings should start with a capital letter and end with full stop.

Reply:
```suggestion
"""Code to output a Fortran module to interface to the given TF models.
```